app/src/main/res/values/styles.xml

```diff
@@ -29,8 +29,10 @@
         <item name="android:paddingEnd">16dp</item>
         <item name="android:textColor">@drawable/primary_button_text</item>
         <item name="android:textSize">16sp</item>
-        <item name="fontFamily">sans-serif-medium</item>
+        <item name="fontFamily">sans-serif</item>
+        <item name="android:textStyle">bold</item>
```

Comment:
This should also be moved to where you added your feature and where exactly you saw this style. I am not sure we want his a the app wide default value.

Reply:
All primary buttons have this textStyle. Isn't that the main use case for styles? Define them once, and apply them in several places.